Dockerfile

```diff
@@ -1,14 +1,22 @@
-FROM golang:1.17.5-alpine3.15 as builder
-RUN apk add --no-cache git ca-certificates make tzdata
-COPY . /app
-RUN cd /app && \
-    go get -d -v && \
-    CGO_ENABLED=0 GOOS=linux go build -v -a -installsuffix cgo -o prometheus_bot
+FROM golang:1.17.6-alpine3.15 as builder
+RUN addgroup -S appgroup && adduser -S appuser -G appgroup -u 1000 \
+  && apk add --update --no-cache ca-certificates tzdata
+WORKDIR /app
+COPY go.mod .
+COPY go.sum .
+RUN go mod download
+COPY . .
+RUN GOGC=off CGO_ENABLED=0 go build -v -o prometheus_bot
 
-FROM alpine:3.14
-COPY --from=builder /app/prometheus_bot /
-RUN apk add --no-cache ca-certificates tzdata tini
-USER nobody
+FROM scratch
+COPY --from=builder /usr/share/zoneinfo /usr/share/zoneinfo
+COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
+COPY --from=builder \
+  /etc/passwd \
+  /etc/group \
+ /etc/
+WORKDIR /
+COPY --from=builder /app/prometheus_bot .
+USER appuser
```

Comment:
How would you like to leave `USER nobody` as it is now and remove all copying `/etc/` stuff?

Reply:
nobody not exist in scratch
You can sheck it.
scratch actually complitely empty image.
And now actually it's running as root.

Reply:
Ah, so `scratch` does not `/etc/passwd` and `/etc/group`, right?

Reply:
Yep, scratch means nothing. It's complitely empty image, I suppose with zero size.